generic/htx_test.py

```diff
@@ -75,7 +75,7 @@ def setUp(self):
         self.mdt_file = self.params.get('mdt_file', default='mdt.mem')
         self.time_limit = int(self.params.get('time_limit', default=2))
         self.time_unit = self.params.get('time_unit', default='m')
-        self.run_type = self.params.get('run_type', default='git')
+        self.run_type = self.params.get('run_type', default='')
```

Comment:
@PraveenPenguin we will have default as `rpm` right?

Reply:
@Naresh-ibm as we don't have logic around on this keyword , as only two way we have git and rpm so i left as blank , what you think